van-gogh-gallery/jonas/2. page/style-collection.css

```diff
@@ -0,0 +1,90 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:wght@400;500&display=swap');
+
+body {
+    height: 100%;
+    width: 100%;
+    margin: 0%;
+    background-image: white
+
+}
+
+
+.headline {
+
+padding-top: 200px;
+padding-left: 200px;
+font-family: 'Barlow Semi Condensed', sans-serif;
+font-size: 5vw;
+}
+
+#container-cards {
+
+padding-top: 200px;
+padding-bottom: 60px;
+padding-left: 15px;
+width: 95%;
+margin-left: 50px;
+display: inline-flex;
+flex-direction: row;
+flex-wrap: wrap;
+align-items: center;
+justify-content: flex-start;
+overflow-x: auto;
+color: ivory;
+
+}
+
+.card {
+
+    width: 300px;
+    height: 300px;
+    margin-right: 60px;
+    margin-top: 60px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-end;
+    flex-direction: row;
+    background-position: center;
+    cursor: pointer;
+    overflow: hidden;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.19);
+    border-radius: 10px;
+    
+}
+
+
+#container-navigation {
+display: flex;
+flex-direction: row;
+justify-content: flex-start;
+justify-content: space-between;
+height: 30px;
+width: 50%;
+margin-top: 50px;
+margin-left: 50px;
+position: fixed;
+
```

Comment:
I would add some background color here to prevent 
![Uploading Screenshot 2021-04-26 at 18.57.15.png…]()